ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -40002,7 +40002,7 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build05
+  cluster: build04
```

Comment:
I was under the impression metal jobs have to route to build05, is this no longer the case, or not true for metal-assisted jobs?

Reply:
I think it's true (or was true, not sure) for ipi-metal jobs. It allows usage of BM machines from internal labs

For assisted-based jobs we're only using Equinix machines